vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -254,6 +254,60 @@ var _ = RecommenderE2eDescribe("VPA CRD object", func() {
 	})
 })
 
+var _ = RecommenderE2eDescribe("VPA CRD object", func() {
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+
+	var (
+		vpaClientSet vpa_clientset.Interface
+	)
+
+	ginkgo.BeforeEach(func() {
+		vpaClientSet = getVpaClientSet(f)
+	})
+
+	ginkgo.It("with all container recommendations", func() {
+		ginkgo.By("Setting up a hamster deployment")
+		d := NewNHamstersDeployment(f, 2 /*number of containers*/)
+		_ = startDeploymentPods(f, d)
+		vpaCRD := createVpaCRDWithContainerScalingMode(f, vpa_types.ContainerScalingModeAuto)
+
+		ginkgo.By("Waiting for recommendation to be filled for both containers")
+		vpa, err := WaitForRecommendationPresent(vpaClientSet, vpaCRD)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(vpa.Status.Recommendation.ContainerRecommendations).Should(gomega.HaveLen(2))
+
+	})
+
+	ginkgo.It("with opt-out container recommendation", func() {
+		ginkgo.By("Setting up a hamster deployment")
+		d := NewNHamstersDeployment(f, 2 /*number of containers*/)
+		_ = startDeploymentPods(f, d)
+		vpaCRD := createVpaCRDWithContainerScalingMode(f, vpa_types.ContainerScalingModeOff)
```

Comment:
It's not clear here that only one container has this scaling mode.
I think either renaming the function (or changing its signature) or adding an explanation here would be helpful.

Reply:
I've changed the function to take trailing arguments and create containers entry based of them.
Comment added.